package.json

```diff
@@ -3,15 +3,20 @@
   "version": "1.0.0",
   "description": "Public website for zinc.coop",
   "main": "index.js",
-  "dependencies": {},
-  "devDependencies": {
-    "gh-pages": "^2.2.0",
-    "ncp": "^2.0.0"
+  "dependencies": {
+    "@11ty/eleventy": "^0.10.0",
+    "bulma": "^0.8.0",
+    "concurrently": "^5.1.0",
+    "node-sass": "^4.13.1",
+    "npm-run-all": "^4.1.5"
   },
   "scripts": {
-    "prebuild": "pandoc src/operating-agreement/index.html -o src/operating-agreement/index.pdf",
-    "build": "ncp src dist",
-    "deploy": "gh-pages --dist dist"
+    "start": "concurrently --kill-others \"npm run css-watch\" \"npm run dev\"",
+    "css-build": "node-sass src/_sass/main.scss dist/css/main.css",
+    "css-watch": "npm run css-build && npm run css-build -- --watch",
```

Comment:
Could we have eleventy do this for us? It seems odd that we need multiple build/watch steps for a static site.